<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wagyu 2.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameIMG">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbIMG">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>66</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Imagens</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbIMG">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameATH">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>201</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelATH">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Atributos de Textura</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameDMCO">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelDMCO">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Distancia MCO</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxDMCO">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameNT">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelNT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de Threads</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxNT">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbExtrair">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>297</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extrair Atributos</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
